### DISCUSSION TITLE
The response from the Google Analytics Data API is empty.

### DISCUSSION THREAD

**Discussion by JayedHasan232** [freekmurze/laravel-analytics#481] [Q&A]
I have carefully followed the installation instructions, but when attempting to use the Google Analytics Data API, I receive an empty response. I would appreciate any insights into the root cause of this issue and any potential solutions that could resolve it.

**Comment by imtayyabhayat** [comment]
it returns empty result when not a valid date is passed.
Make sure you have imported this:

`use Spatie\Analytics\Period;`

as it is not mentioned in instruction file. Verify it by using  `dd(Period::days(7);`
